forte/processors/data_augment/algorithms/back_translation_op.py

```diff
@@ -120,7 +107,41 @@ def replace(self, input_anno: Annotation) -> Tuple[bool, str]:
             replaced string.
         """
         # If the replacement does not happen, return False.
-        if random.random() > self.configs.prob:
+        if random.random() > self.configs["prob"]:
             return False, input_anno.text
         intermediate_text: str = self.model_to.translate(input_anno.text)
         return True, self.model_back.translate(intermediate_text)
+
+    @classmethod
+    def default_configs(cls) -> Dict[str, Any]:
+        """
+        Returns:
+            A dictionary with the default config for this processor.
+        Following are the keys for this dictionary:
+            - `augment_entry` (str): This indicates the entity that needs to
+                be augmented. By default, this value is set to
```

Comment:
Take a look at the generated documentation for this part: https://asyml-forte--717.org.readthedocs.build/en/717/code/data_aug.html#forte.processors.data_augment.algorithms.back_translation_op.BackTranslationOp.default_configs

The format is weird, mainly because of indentation.

Reply:
Also, take a look at the doc rendering on this web page, and fix the others too.